target/linux/bmips/dts/bcm6328-dlink-dsl-2750b-b1.dts

```diff
@@ -0,0 +1,276 @@
+#include "bcm6328.dtsi"
+
+/ {
+	model = "D-Link DSL-2750B rev B1/DSL-2740B rev F1/DSL-2741B rev F1";
+	compatible = "dlink,dsl-2750b-b1", "brcm,bcm6328";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_green;
```

Comment:
led_power_red?

Reply:
I was unsure if it should be used. 
fixed, removed as not used.